pkg/proxy/proxy_test.go

```diff
@@ -73,7 +73,7 @@ func TestListAllReleases(t *testing.T) {
 	proxy := newFakeProxy([]AppOverview{app1, app2})
 
 	// Should return all the releases if no namespace is given
-	releases, err := proxy.ListReleases("", 256)
+	releases, err := proxy.ListReleases("", 256, "")
```

Comment:
Should we have a test case that filters through some releases here?

Reply:
I couldn't do that because the fake helm client ignores most of the filters: https://github.com/helm/helm/blob/master/pkg/helm/fake.go#L61 (that's what I tried to explain the TODO below. I couldn't find a way of at least checking that we are passing the correct parameters to helm.